src/CoreDAL/CoreDAL.csproj

```diff
@@ -7,6 +7,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.EntityFrameworkCore" Version="5.0.14" />
     <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="5.0.14" />
+    <PackageReference Include="Z.EntityFramework.Extensions.EFCore" Version="5.13.10" />
```

Comment:
I'm sure that solution should exists without this package.

Reply:
done